### PR TITLE
added Umea meetup

### PR DESCRIPTION
**Pull request by tordsson**
Added info about all events to date for the Umea meetup group.

### CODE REVIEW

File: umea/meetup.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+city: Umeå
+country: sweden
+meetupID: Cloud-Native-Northern-Sweden
+meetups:
+- address: Tvistevägen 47
+  attendees: 19
+  date: "2018-05-28T17:00:00Z"
+  duration: 2h0m0s
+  id: 261450080
+  name: Tremds in cloud native - a summary from KubeCon + CloudNativeCon Barcelona
```
Comment (luxas) [inline umea/meetup.yaml]: typo: `trends`

File: umea/meetup.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  presentations:
+  - endTime: "18:30"
+    slides: null
```
Comment (luxas) [inline umea/meetup.yaml]: can you upload slides here?

File: umea/meetup.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    speakers: 
+      - Linda Almersson
```
Comment (luxas) [inline umea/meetup.yaml]: for every speaker and company mentioned here, can you append these to `companies.yaml` and `speakers.yaml` top-level in the repo?

See e.g. #56 of how to do this. Sorry, we're lacking documentation of this I know

File: umea/meetup.yaml (hunk 1 above)
Comment (luxas) [reply umea/meetup.yaml]: ah right this is probably coming directly from the meetup API and can't be changed... ok

File: tampere/meetup.yaml [hunk 4]
```diff
@@ -129,7 +129,7 @@ meetups:
     - luxaslabs
     - cncf
     venue: cybercom
-members: 150
```
Comment (luxas) [inline tampere/meetup.yaml]: please drop this change, I fixed the cause of this being an issue at latest master

File: umeå/meetup.yaml [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  id: 261450080
+  name: 'Trends in cloud native - a summary from KubeCon + CloudNativeCon Barcelona '
+  presentations:
+  - endTime: "18:30"
+    slides: ""
+    speakers:
+    - lindaalmersson
+    - tordsson
+    startTime: "17:00"
+    title: ""
```
Comment (luxas) [inline umeå/meetup.yaml]: duplicate the title here too?

File: umeå/meetup.yaml [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    other: null
+    venue: elastisys
+- address: Västra Norrlandsgatan 10D
+  attendees: 36
+  date: "2019-05-07T17:00:00Z"
+  duration: 2h0m0s
+  id: 260276456
+  name: What does “production ready” really mean for a Kubernetes cluster?
+  presentations:
+  - endTime: "18:30"
```
Comment (luxas) [inline umeå/meetup.yaml]: I fixed endTime/startTime at latest master, now just put `delay: 30m0s` and `duration: 60m0s` instead (for all presentations)

File: umeå/meetup.yaml [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+    slides: https://speakerdeck.com/luxas/intro-to-the-cloud-native-world-of-kubernetes-november-2018
+    speakers:
+    - luxas
+    startTime: "17:30"
+    title: Intro to the cloud-native world with Kubernetes
+  sponsors:
+    other: null
+    venue: elastisys
+members: 128
```
Comment (luxas) [inline umeå/meetup.yaml]: this part has been dropped from master, sorry. remove it and re-run the generator

File: umea/meetup.yaml (hunk 3 above)
Comment (tordsson) [reply umea/meetup.yaml]: Done in latest commit, I'll rebase

**Comment by tordsson** [comment]
Thanks for the review! I'll rebase and recommit the next few days.

**Comment by kaspernissen** [comment]
@tordsson let us know if there's anything we can do to help you get this merged. 👍

**Pull request merged**